content/bn/auto_scaling.md

```diff
@@ -0,0 +1,11 @@
+---
+title: অটোস্কেলিং (Auto Scaling)
```

Comment:
Can we have the title as `অটোস্কেলিং(Autoscaling)` ? For reference kindly check this [Glossary Page link](https://glossary.cncf.io/auto_scaling/)